Auto load now works with custom blender extensions folders

RinaDev-xyz opened the pull request:
- Enable autoload to load things from outside the default blender folder, before is a user tried to install the plugin outside from blender default extension folder it will error. I found this out when i broke my own blender helping someone and has to add my own custom folder.

Merge whenever.

RinaDev-xyz committed:
Auto load now works with custom blender extensions folders

- Enable autoload, load things from outside the default blender folder, before is a user tried to install the plugin outside from blender default extension folder it will error. I found this out when i broke my own blender helping someone.

989onan commented:
Uhhh is using System path changes absolutely nessiary? Or does blenders python not exposed enough to let it find the file via some other means?

If the sys.path appending is unavoidable I can merge, but I feel like if Avatar Toolkit goes on the future blender add-ons site they might get pissy about it heh

RinaDev-xyz committed:
Let's not use System path

RinaDev-xyz commented:
This should be a better way which seems to work just fine.

At the time this way, which i was trying to get to work was crashing blender, so I did sys path as a temp dirty solution unto i read the docs on blender API however i soon realize what I was doing wrong which most likely was causing the crash (Though it should be noted blender shouldn't of crashed, but blender likes to crash on werid things). This way is more simple and should be better and I hope you agree.

RinaDev-xyz committed:
I don't need to add that check duh

RinaDev-xyz committed:
Fix because i stupid

RinaDev-xyz committed:
Remove wheel installation

- Blender should handle this for us, we were installing system wide which is bad.

989onan commented:
This is a much better solution thanks! I just feel like people may get "do you want blender to make changes to your system" and that would make almost everyone raise red flags lol!

Merging
